src/main/java/com/example/finance/controller/BudgetController.java

```diff
@@ -38,8 +38,8 @@ public ResponseEntity<BudgetDto> create(@RequestBody BudgetDto budgetDto) {
     }
 
     @PutMapping(value = ApiEndpoints.Endpoints.ID)
-    public ResponseEntity<BudgetDto> update(@PathVariable UUID id, @RequestBody BudgetDto budget) {
-        BudgetDto budgetDto = budgetService.updateBudget(id, budget);
+    public ResponseEntity<BudgetDto> update(@RequestBody BudgetDto budget) {
+        BudgetDto budgetDto = budgetService.updateBudget(budget);
         return ResponseEntity.ok(budgetDto);
```

Comment:
You can put it in one line, because you does not do anything with `budgetDto` :)